### DISCUSSION TITLE
Welcome to Reactive Resume Discussions!

### DISCUSSION THREAD

**Discussion by amruthpillai** [AmruthPillai/Reactive-Resume#692] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us how you use Reactive Resume, or how it has helped you. It always great to start off with some positivity :)

**Comment by anseenivasan** [comment]
the Cv shows all content in one Page and doesn't move to page 2 once there is break line

**Reply by amruthpillai** [reply]
You need to move it to another page. It's not automatic.

**Comment by martadinata666** [comment]
about env and generated js, is the generated js work like v2, the env become embedded to js? So i  need to rebuild everytime env file change?

**Reply by amruthpillai** [reply]
No, you don't need to rebuild every time the `.env` file changes. It's pulled in at runtime, so it's possible to change them and just restart the server.

**Comment by martadinata666** [comment]
another question 🤣 , sorry keep bothering you. 
So after play around with `pnpm dev` and add simple nginx reverse proxy and work quite well, then i do `pnpm build`, then `pnpm start`. Somehow the client missing access to `localhost:3100` is this intended? Aafter, poking around the codes and i realized i need to add `NODE_ENV=development` and do `pnpm build` again to force client route to localhost:3100 again.

**Comment by StarHamster** [comment]
When I copy the link to my resume using the button in the artboard controller (at the bottom of the page), the full link is copied even if the "Prefer Short URL" option is on. Is that ok?
